backend/src/main/java/com/wpz/rbs/service/RoomService.java

```diff
@@ -17,9 +19,15 @@ public RoomService(RoomRepository roomRepository) {
         this.roomRepository = roomRepository;
     }
 
-    public List<Room> getAll() {
+    public List<Room> getAllFiltered(String number, String type, Integer capacity, String annotation) {
+        var exampleRoom = new Room();
+        exampleRoom.setNumber(number);
+        exampleRoom.setType(type);
+        exampleRoom.setCapacity(capacity);
```

Comment:
I think that we should return rooms with capacity equal or grater than provided by user.